source/overloading.tex

```diff
@@ -4259,7 +4259,7 @@
 float operator ""_e(const char*);                   // OK
 float operator ""E(const char*);                    // error: reserved literal suffix~(\ref{usrlit.suffix}, \ref{lex.ext})
 double operator""_Bq(long double);                  // OK: does not use the reserved identifier \tcode{_Bq}\iref{lex.name}
-double operator"" _Bq(long double);                 // uses the reserved identifier \tcode{_Bq}\iref{lex.name}
+double operator"" _Bq(long double);                 // error: uses the reserved identifier \tcode{_Bq}\iref{lex.name}
```

Comment:
Do we say "error:" for "no diagnostic required" cases?

Reply:
At least, we do two lines earlier for `""E`.

In other cases, we say "ill-formed, no diagnostic required" in the example comment.